### ISSUE TITLE
Replace all `template_tag` with `div`

### ISSUE THREAD

**Issue by SethHorsley** [rbui-labs/rbui#159]
Template tags are causing issues with connecting stimulus into the template tag. So it would be best to remove it.

**Comment by djalmaaraujo** [comment]
I checked the current codebase and there are no remaining `template_tag` calls, but there are still HTML `<template>` usages in components such as Dialog, Sheet, Command, AlertDialog, ContextMenu, HoverCard, and Calendar templates. Is the intended scope of this issue to remove all rendered `<template>` tags as well, or only the older `template_tag` helper usage?

**Comment by djalmaaraujo** [comment]
Closing this this is from 2024

**Comment by sethhorsley** [comment]
Yeah I think it would be best to remove the <template> tags from dialog, sheet, command, etc…
